gtsfm/densify/mvs_base.py

```diff
@@ -14,10 +14,15 @@
 from gtsfm.common.gtsfm_data import GtsfmData
 from gtsfm.evaluation.metrics import GtsfmMetricsGroup
 
+from gtsfm.ui.registry import GTSFMProcess, UiMetadata
 
-class MVSBase(metaclass=abc.ABCMeta):
+
+class MVSBase(GTSFMProcess):
     """Base class for all multi-view stereo implementations."""
 
+    def get_ui_metadata() -> UiMetadata:
+        return UiMetadata("Multi-view Stereo", ("Images", "SFM Result"), ("Dense Points"), "")
```

Comment:
Do you mind adding a 1-line docstring here explaining what is being returned here?